dojos.logic

```diff
@@ -0,0 +1,12 @@
+area Dojos:
+  room Ghostblade
+  room Grayblade
+  room Greatblade
+  room Grimblade
+  room Spinblade
```

Comment:
Who is spinblade? Did you mean scarblade (in castor)?

Reply:
Oh, i just based the names of what minishmaker calls them, but yes scarblade is the proper name, ill change it